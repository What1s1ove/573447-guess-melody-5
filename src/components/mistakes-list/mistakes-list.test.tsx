import * as React from 'react';
import { render } from 'enzyme';
import MistakesList from './mistakes-list';

describe(`MistakesList component`, () => {
  it(`should render MistakesList component`, () => {
    const component = render(<MistakesList mistakesCount={1} />);

    expect(component).toMatchSnapshot();
  });
});
