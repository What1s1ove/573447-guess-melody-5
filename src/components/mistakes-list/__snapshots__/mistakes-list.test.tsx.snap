// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MistakesList component should render MistakesList component 1`] = `
<div
  class="game__mistakes"
>
  <div
    class="wrong"
  />
</div>
`;
